GTFS.DB.SQLite/SQLiteGTFSFeedDB.cs

```diff
@@ -220,6 +220,8 @@ private void RebuildDB()
             this.ExecuteNonQuery("CREATE TABLE IF NOT EXISTS [cleaned_stops] ( [stop_id] TEXT);");
             // CREATE TABLE TO STORE LOGS OF EDITS
             this.ExecuteNonQuery("CREATE TABLE IF NOT EXISTS [log] ( [timestamp] TEXT, [action] TEXT, [route_id] TEXT, [pc_name] TEXT, [pc_ip] TEXT, [note] TEXT);");
+            // CREATE TABLE TO STORE POLYGONS
+            this.ExecuteNonQuery("CREATE TABLE IF NOT EXISTS [polygons] ( [id] TEXT NOT NULL, [poly_pt_lat] REAL, [poly_pt_lon] REAL, [poly_pt_seq] INTEGER );");
```

Comment:
I see we are creating the table here which is correct, do you not need to add other classed to access the info? Or how are you pulling the polygons into capture?

Reply:
It's all in GTFSHelpers. It's very reliant on the GMapPolygon object in GMap. Didn't want to add the entire GMap.Net package to GTFS.

Reply:
Yeah that's fine